migrations/Version202606101250452141_taoItems.php

```
@@ -0,0 +1,39 @@
+        $script = new SetupSectionVisibilityFilters();
+        $script->setServiceLocator($this->getServiceLocator());
+        $script([]);
+
+        $this->addReport(
+            Report::createSuccess(
+                'Usage and Item Statistics actions are now controlled by FEATURE_FLAG_MANAGE_ITEMS'
+            )
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)